src/app/app-routing.module.ts

```diff
@@ -15,6 +16,8 @@ const routes: Routes = [
     path: 'registration', component: RegistrationComponent
   }, {
     path: 'signIn', component: SignInComponent
+  }, {
+    path: 'navbar', component: NavbarComponent
```

Comment:
Mhhh if I understand that right, this path is confusing is the Navbar not a part of a Page and should be part of a different path?
Maybe this could give you a hint in the right direction:
https://www.techiediaries.com/angular-material-navigation-toolbar-sidenav/